tests/test_playbooks/content_import_info.yml

```diff
@@ -0,0 +1,66 @@
+---
+- hosts: foreman
+  remote_user: root
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - name: Extract import tgz
+      ansible.builtin.unarchive:
+        src: data/content-import.tgz
+        dest: /var/lib/pulp/imports
+        owner: pulp
+        group: pulp
+
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - name: Setup Org
+      include_tasks: tasks/organization.yml
+      vars:
+        organization_state: present
+    - name: Test Import Repository
+      include_tasks: tasks/content_import.yml # Full export
+      vars:
+        import_type: "repository"
+        expected_change: true
```

Comment:
we don't expect a change here (it will happen, but we shall not expect it, it's the setup phase)